src/components/TableSkeletonLoader.js

```diff
@@ -0,0 +1,38 @@
+import { div } from 'react-hyperscript-helpers';
+import { Styles } from '../libs/theme';
+
+export default function DarTableSkeletonLoader(props) {
```

Comment:
nit: discrepancy between function and file name. Since this is supposed to be generic, it should probably be `TableSkeletonLoader`. That being said, `TableSkeletonLoader` is a little obscure of a name for me. I don't have a better suggestion, and not a reason to hold up this PR, but something that more conveys the temporary spinning nature might be helpful. Again, super minor nit and don't let this hold up the PR.

Reply:
Whoops, forgot to update the function name. That said, these things are known as skeleton loaders or skeleton screens with most UI or UX experts, so I'm just following their naming convention.

Reply:
Works for me ... I can try and learn a new trick.